spin/cmds/meson.py

```diff
@@ -109,8 +109,15 @@ def _meson_version_configured():
 @click.option(
     "-v", "--verbose", is_flag=True, help="Print detailed build and installation output"
 )
+@click.option(
+    "--gcov-build",
```

Comment:
For flags, I think recommended naming is `--gcov/--no-gcov`; see https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags.

Reply:
+1 for `--gcov` as the name. I don't see a point in adding `--no-gcov`, that doesn't do anything and is extra noise in `--help`. So I'd go for the second option documented in those Click docs under "If you really don’t want an off-switch".

Reply:
That works!

Reply:
Added via [20276fc](https://github.com/scientific-python/spin/pull/146/commits/20276fc80612ca5a30c7de05c8c9eeac829c2eae)